llvm/lib/Target/AIE/AIE2InstrInfo.cpp

```diff
@@ -1158,392 +1159,6 @@ AIE2InstrInfo::getTiedRegInfo(unsigned Opcode) const {
   }
 }
 
-std::optional<unsigned>
-AIE2InstrInfo::getOpcodeWithAtomicOperands(unsigned Opcode) const {
-  // TODO: This should be tablegen-erated.
```

Comment:
WELLDONE: This has become tablegen-generated

Reply:
This was my masterplan all along 😈